backstop/tests/carousel.html

```diff
@@ -0,0 +1,270 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>Carousel | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      section[id^='demo'] {
+        margin-bottom: 22px;
+      }
+
+      /* These dimensions equal 16.9 ratio */
+      .demo-carousel-width {
+        width: 600px;
+      }
+
+      .demo-carousel-height {
+        height: 337.5px;
+      }
+
+      /* Custom content within slide 1 */
+      .demo-columns {
+        display: flex;
+        width: 100%;
+        height: 100%;
+        background-color: var(--iui-color-background-4);
+      }
+
+      .demo-column {
+        width: 50%;
+        height: 100%;
+        box-sizing: border-box;
+      }
+
+      .demo-left {
+        padding: 11px 12px;
+      }
+
+      .demo-right {
+        background-image: linear-gradient(to right, var(--iui-color-background-4), rgba(var(--iui-color-background-4-rgb), 0)), url('../assets/image-test.jpg');
+        background-position: center;
+        background-size: cover;
+      }
+    </style>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/chevron-left"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/chevron-right"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/pause"
+    ></script>
+    <script
+      type="module"
+      src="https://cdn.skypack.dev/@itwin/itwinui-icons-elements/play"
```

Comment:
Not used

Reply:
It is now. 😆 The play / pause button will toggle when clicked (although it appears the button width is being effected because technically there are 2 elements within the icon, right click and inspect the margin on the icon compared to the next / previous buttons).

Reply:
Oh wow, I didn't realize we were using `:only-child` in our CSS.

Updated code in [`bbccbfd`](https://github.com/iTwin/iTwinUI/pull/377/commits/bbccbfdf596c95c89f3e05210e4fce42873c2bb7) to only have one SVG in the button.

Reply:
amm, yes, clicking the button toggles between play and pause, but technically it does nothing. What are we trying to show here?
Id say just leave single element if we want to show that any element can be put additionaly :/